apps/storybook/src/Surface.stories.tsx

```diff
@@ -39,3 +50,180 @@ export const Basic: Story<SurfaceProps> = ({ elevation }) => {
+  const cardStyle = {
+    maxHeight: '300px',
+  };
+  return (
+    <Surface
+      elevation={elevation}
+      style={cardStyle}
+      layout={layout}
+      padded={padded}
+    >
+      <Surface.Header>
+        <Text variant='subheading' as='h2'>
+          Surface with overflow & button footer
+        </Text>
+      </Surface.Header>
+      <Surface.Body>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+          culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum
+          dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+          quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+          commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+          velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+          occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+          mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur
+          adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+          ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+          irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+          fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+          sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem
+          ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+          veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+          ea commodo consequat. Duis aute irure dolor in reprehenderit in
+          voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+          sint occaecat cupidatat non proident, sunt in culpa qui officia
+          deserunt mollit anim id est laborum.
+        </p>
+      </Surface.Body>
+      <Divider />
+      <Button styleType='borderless'>View All</Button>
+    </Surface>
+  );
+};
+
+Overflow.args = { elevation: 1, layout: true, padded: true };
+
+export const NoPadding: Story<SurfaceProps> = (args) => {
+  const { elevation, layout, padded } = args;
+
+  const cardStyle = {
+    maxHeight: '300px',
+  };
+  return (
+    <Surface
+      elevation={elevation}
+      style={cardStyle}
+      layout={layout}
+      padded={padded}
+    >
+      <style>
```

Comment:
I've seen styling added this way in the `CenterContent` story in `Header` [here](https://github.com/iTwin/iTwinUI/blob/main/apps/storybook/src/Header.stories.tsx#L279)